ClientApp/viking-vault/src/ProfilePicture.css

```diff
@@ -2,17 +2,16 @@
 {
     position: relative;
     float: left;
```

Comment:
It would be nice if you didn't use `float` as they are a bit nasty because of the height problem of the container when you use floats.